fedimint-core/src/module/mod.rs

```diff
@@ -881,12 +881,6 @@ pub trait ServerModule: Debug + Sized {
     ///
     /// Depending on the module this might contain data needed by the client to
     /// access funds or give an estimate of when funds will be available.
-    ///
```

Comment:
We should probably still give the correct guidance on the meaning of `None`, even if just to document the current state for our own purposes.